opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,64 @@
+package nl.novi.opdrachttechiteasy.controllers;
+
+
+import nl.novi.opdrachttechiteasy.models.Televisions;
+import nl.novi.opdrachttechiteasy.repositories.TelevisionRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RequestMapping("/televisions")
+@RestController
+public class TelevisionController {
+
+    private final TelevisionRepository televisionRepository;
+
+    public TelevisionController(TelevisionRepository televisionRepository) {
+        this.televisionRepository = televisionRepository;
+    }
+
+
+    @GetMapping()
+    public ResponseEntity<List<Televisions>> television(){
+        return ResponseEntity.ok(televisionRepository.findAll());
+    }
+
+    @GetMapping("/type")
+    public ResponseEntity<String> television(@RequestParam String type){
+        return ResponseEntity.ok("Television " + type);
```

Comment:
Deze heb je nog niet met de repository gedaan. Je hebt in de repository al wel een "finByBrand" methode en zo kun je ook een "findByType" methode maken om hier te gebruiken.